config/serverless.security.search_ai_lake.yml

```diff
@@ -7,8 +7,9 @@ xpack.features.overrides:
   ### The following features are Security features hidden in Role management UI for this specific tier.
   securitySolutionTimeline.hidden: true
   securitySolutionNotes.hidden: true
+  securitySolutionSiemMigrations.hidden: true
```

Comment:
Do we care if the user is able to run siem_migrations? or do we only want to pretend the feature does not exist in the roles UI?